tests/Issue.py

```diff
@@ -72,6 +72,7 @@ def testAttributes(self):
         self.assertEqual(self.issue.pull_request.patch_url, None)
         self.assertEqual(self.issue.pull_request.html_url, None)
         self.assertEqual(self.issue.state, "closed")
+        self.assertEqual(self.issue.state_reason, "completed")
```

Comment:
While you are there, can you please add tests for the following fields, ideally in the order of their definition in `_initAttributes`?

- comments_url after testing comments
- events_url after testing created_at
- labels_url after testing labels

Reply:
Hi @EnricoMi ok I'll do that

Reply:
Sorry for being sooo long, I've added what you've asked I guess @EnricoMi.
Ping @Thunderforge, I guess it could be merged soon now

Reply:
Looks good to me.

@EnricoMi Are you okay with these improvements?